api/s2n.h

```diff
@@ -144,6 +144,7 @@ typedef enum { S2N_BUILT_IN_BLINDING, S2N_SELF_SERVICE_BLINDING } s2n_blinding;
 extern int s2n_connection_set_blinding(struct s2n_connection *conn, s2n_blinding blinding);
 extern uint64_t s2n_connection_get_delay(struct s2n_connection *conn);
 
+extern int s2n_conn_set_cipher_preferences(struct s2n_connection *conn, const char *version);
```

Comment:
a naming nit: The other functions that use the `struct s2n_connection` start with `s2n_connection_*`. Should we use `s2n_connection_set_cipher_preferences` instead?